imports/ui/Login.css

```
@@ -0,0 +1,57 @@
+.loginForm {
+  width: 493px;
+  height: 625px;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 25px;
+}
+
+.formTitle {
+  color: black;
+  text-align: center;
+}
+
+.inputGroup {
+  border-radius: 25px;
+  margin-left: auto;
+  margin-right: auto;
+  background: #e8e8e8;
+  border: none;
+}
+
+.logoContainer {
+  border: none;
+}
+
+.inputContainer {
+  width: 396px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 35px;
+  margin-bottom: 35px;
+}
+
+.loginInput {
```

When I ran it locally, the placeholder text looks small. Check the Figma specs for how large the font should be in the input. Both the placeholder text, and regular text should be the same size. 

Here's a good reference for styling placeholder text: https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder

---

It would be something like:

```suggestion
.loginInput::placeholder {
  // your-styles-here: dope-styles;
 }
```